service/src/main/java/org/mskcc/cmo/metadb/service/SampleService.java

```diff
@@ -19,7 +19,8 @@
 
     MetaDbSample getMetaDbSample(UUID metaDbSampleId) throws Exception;
 
-    MetaDbSample getMetaDbSampleByRequestAndIgoId(String requestId, SampleAlias igoId) throws Exception;
+    MetaDbSample getMetaDbSampleByRequestAndIgoSampleAlias(String requestId,
```

Comment:
Can we drop Igo from the method name?

Reply:
maybe `getMetaDbSampleByRequestAndIgoSampleAlias` --> `getMetaDbSampleByRequestAndAlias` ?